app/src/main/java/cm/aptoide/pt/appview/OtherVersionsView.kt

```diff
@@ -32,13 +30,14 @@ import cm.aptoide.pt.feature_apps.presentation.AppsListUiState
 import cm.aptoide.pt.feature_apps.presentation.appVersions
 
 @Composable
-fun OtherVersionsView(packageName: String, listScope: LazyListScope?) {
+fun OtherVersionsView(packageName: String) {
   val (uiState, _) = appVersions(packageName = packageName)
-
-  listScope?.item { Box(modifier = Modifier.padding(top = 26.dp)) }
-  (uiState as? AppsListUiState.Idle)?.run {
-    listScope?.items(apps) { app ->
-      OtherVersionRow(app)
+  Column {
+    Box(modifier = Modifier.padding(top = 24.dp))
```

Comment:
This padding can be applied to the parent Column, instead of creating a new Box. An alternative would also be to use Spacer instead of Box